packages/atomic/src/components/result-lists/result-list-common.pcss

```diff
@@ -73,9 +73,18 @@
   }
 }
 
-a[part='result-list-grid-clickable'] {
-  display: grid;
-  text-decoration: none;
+[part='result-list-grid-parent'] {
+  position: relative;
+  cursor: pointer;
+}
+
+[part='result-list-grid-clickable'] {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  pointer-events: none;
```

Comment:
doesn't this prevent the click event from being dispatched?

Reply:
🤦 🤦 🤦 🤦 🤦

Reply:
Reworking lol